config/environments/development.rb

```diff
@@ -30,8 +30,7 @@
   config.action_mailer.raise_delivery_errors = false
 
   # MailCatcher
-  config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = { address: "localhost", port: 1025 }
```

Comment:
Can be removed from development - we should still default to localhost:1025 for [mailcatcher](https://mailcatcher.me/)